MoreJS/es6-features.md

```diff
@@ -1,6 +1,7 @@
 I am hoping that we will not have to cover too much here.. Presumably many of the syntax things and features will be covered in earlier lessons.. however there are a few little details that need covered, including a section on transpilers.
 things to cover include:
-- Babel! (maybe we should write our own.. but this tutorial is pretty hip https://github.com/verekia/js-stack-from-scratch)
+- Babel! (maybe we should write our own.. but this tutorial is pretty hip https://github.com/verekia/js-stack-from-scratch)  
+	- babel's own site is pretty good.  their live demo is great for getting the point
```

Comment:
the point is easy enough... the main thing the lesson should cover is how to use it and integrate it into the workflow.